Lib/test/test_venv.py

```diff
@@ -531,7 +531,8 @@ def test_multiprocessing_recursion(self):
         rmtree(self.env_dir)
         self.run_with_capture(venv.create, self.env_dir)
         script = os.path.join(TEST_HOME_DIR, '_test_venv_multiprocessing.py')
-        subprocess.check_call([self.envpy(real_env_dir=True), script])
+        subprocess.check_call([self.envpy(real_env_dir=True), script],
+                              env={**os.environ, "PYTHONHOME": ""})
```

Comment:
Only setting PYTHONHOME sounds a little bit too specific. I suggest to use the isolated option instead, ``-I``:

```suggestion
        subprocess.check_call([self.envpy(real_env_dir=True), "-I", script])
```

Reply:
Thanks for the review! I test it locally and it can resolve the issue too.